hack/manual-testing.sh

```diff
@@ -13,25 +13,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ -z "${REGISTRY}" ] || [ -z "${MANAGER_IMAGE_TAG}" ]; then
-    echo "please set environment variables REGISTRY and MANAGER_IMAGE_TAG"
-    exit 1
+# Verify the required Environment Variables are present.
+: "${AZURE_SUBSCRIPTION_ID:?Environment variable empty or not defined.}"
+: "${AZURE_TENANT_ID:?Environment variable empty or not defined.}"
+: "${AZURE_CLIENT_ID:?Environment variable empty or not defined.}"
+: "${AZURE_CLIENT_SECRET:?Environment variable empty or not defined.}"
+: "${REGISTRY:?Environment variable empty or not defined.}"
+
+# Cluster settings.
+export CLUSTER_NAME="${CLUSTER_NAME:-capz-test}"
+export AZURE_VNET_NAME=${CLUSTER_NAME}-vnet
+
+# Azure settings.
+export AZURE_LOCATION="${AZURE_LOCATION:-southcentralus}"
+export AZURE_RESOURCE_GROUP=${CLUSTER_NAME}
+export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 | tr -d '\n')"
+export AZURE_TENANT_ID_B64="$(echo -n "$AZURE_TENANT_ID" | base64 | tr -d '\n')"
+export AZURE_CLIENT_ID_B64="$(echo -n "$AZURE_CLIENT_ID" | base64 | tr -d '\n')"
+export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -d '\n')"
+
+# Machine settings.
+export CONTROL_PLANE_MACHINE_COUNT=${CONTROL_PLANE_MACHINE_COUNT:-3}
+export AZURE_CONTROL_PLANE_MACHINE_TYPE="${CONTROL_PLANE_MACHINE_TYPE:-Standard_D2s_v3}"
+export AZURE_NODE_MACHINE_TYPE="${NODE_MACHINE_TYPE:-Standard_D2s_v3}"
+export WORKER_MACHINE_COUNT=${WORKER_MACHINE_COUNT:-2}
+export KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.17.3}"
+
+# Generate SSH key.
+SSH_KEY_FILE=${SSH_KEY_FILE:-""}
+if ! [ -n "$SSH_KEY_FILE" ]; then
+    SSH_KEY_FILE=.sshkey
+    rm -f "${SSH_KEY_FILE}" 2>/dev/null
+    ssh-keygen -t rsa -b 2048 -f "${SSH_KEY_FILE}" -N '' 1>/dev/null
+    echo "Machine SSH key generated in ${SSH_KEY_FILE}"
 fi
+export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
 
 echo "================ DOCKER BUILD ==============="
 make docker-build
-echo "================ DOCKER PUSH ==============="
-make docker-push
- 
+
 echo "================ MAKE CLEAN ==============="
 make clean
- 
-echo "================ MAKE MANIFESTS ==============="
-make generate-manifests
-echo "================ MAKE BINARIES ==============="
-make binaries
```

Comment:
Are these no longer needed?

Reply:
no there aren't needed since we moved to v1alpha3